app/src/main/res/values/strings.xml

```diff
@@ -106,7 +106,7 @@
     <string name="q7">Avoids specific things (Specific phobia)</string>
     <string name="q8">Repetitive problematic behaviours (Habit problems)</string>
     <string name="q9">Depression/low mood (Depression)</string>
-    <string name="q10">Self-Harm (Self-injury or self-harm)</string>
+    <string name="q10"> Self-Harm (Self-injury or self-harm)</string>
```

Comment:
I was trying out for the layout at first for some lines and should delete all those, but missed one :P